### ISSUE TITLE
New feature---Tidal Lane??

### ISSUE THREAD

**Issue by pcfantasy** [krzychu124/Cities-Skylines-Traffic-Manager-President-Edition#122]
Is it possible to implemente a tidal lane function in TMPE.
    Let`s talk about some details in this topic.

**Comment by aubergine10** [comment]
What is a tidal lane?

**Comment by aubergine10** [comment]
Something like this:

* https://en.wikipedia.org/wiki/Reversible_lane

Where lane direction can change depending on certain conditions?

**Comment by pcfantasy** [comment]
@aubergine10 

Yes, that`s what I want.

With RealTime, in the morning and afternoon, there will be totally different direction for traffic flow.
Reversible lane is very useful in that case.

**Comment by aubergine10** [comment]
Yes, I was also thinking of RealTime mod by @dymanoid - it would be perfect feature for rush hour traffic.

In addition, if we could wire in different 'programmes' for timed traffic lights, or making them somehow handle the lane direction switching....  Tagging #8

**Comment by aubergine10** [comment]
One way to implement this could be to flip the direction of an asymetric road, although it might look a bit weird.

```
---->
---->
<----
```

becomes:

```
---->
<----
<----
```

**Comment by aubergine10** [comment]
Or those roads with pseudo-bidirectional center lane (usually two lanes, one on top of the other, going different directions) could have one of the lanes disabled.

```
---->
<--->
<----
```

Becomes:

```
---->
---->
<----
```

And can be flipped to:

```
---->
<----
<----
```

**Comment by Monniasza** [comment]
This will need new tools called
"Demand-sensitive lanes",
"Time dependent lane directions",
"Time dependent speed limits",

**Comment by aubergine10** [comment]
@Monniasza Yes, it would probably need quite a bit of new stuff as there are many ideas for time-related customisations: https://github.com/krzychu124/Cities-Skylines-Traffic-Manager-President-Edition/milestone/11
